<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кредитный калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(45, 42, 55)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>166</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>Тип ежемесячных платежей</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>30</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_amount">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>31</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
    color: rgb(255, 255, 255); 
qproperty-alignment: 'AlignVCenter | AlignRight';
background-color:rgb(80, 75, 98);
}



</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_month">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>76</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
    color: rgb(255, 255, 255); 
qproperty-alignment: 'AlignVCenter | AlignRight';
background-color:rgb(80, 75, 98);
}



</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white;
}</string>
   </property>
   <property name="text">
    <string>Срок кредита (месяцы)</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_rate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
    color: rgb(255, 255, 255); 
qproperty-alignment: 'AlignVCenter | AlignRight';
background-color:rgb(80, 75, 98);
}



</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>129</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: white;
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>Ежемесячный платёж</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>270</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">Line {
color: white;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>408</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>Долг+проценты</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthlypayment">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    color: rgb(255, 255, 255); 
qproperty-alignment: 'AlignVCenter | AlignRight';
background-color:rgb(80, 75, 98);
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_accruedinterest">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>359</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    color: rgb(255, 255, 255); 
qproperty-alignment: 'AlignVCenter | AlignRight';
background-color:rgb(80, 75, 98);
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_debtandinterest">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    color: rgb(255, 255, 255); 
qproperty-alignment: 'AlignVCenter | AlignRight';
background-color:rgb(80, 75, 98);
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_count">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border:  3x;
    border-radius:  15px; 
    background-color: qlineargradient(spread:pad, x1:0.5, y1:0.943, x2:0.584, y2:0, stop:0 rgba(50, 34, 93, 255), stop:1 rgba(102, 40, 162, 255));
    color: rgb(255,  255,  255);
    padding:  5px;
    font: bold  15px;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.5); </string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>469</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border:  3x;
    border-radius:  15px; 
    background-color: qlineargradient(spread:pad, x1:0.5, y1:0.943, x2:0.584, y2:0, stop:0 rgba(50, 34, 93, 255), stop:1 rgba(102, 40, 162, 255));
    color: rgb(255,  255,  255);
    padding:  5px;
    font: bold  15px;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.5); </string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>201</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_annuity">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: white;
}

QRadioButton::indicator {
    width: 10px;
    height: 10px;
    border: 2px solid white; /* цвет рамки */
    background-color: #333333; /* цвет фона индикатора */
    border-radius: 5px; /* скругление углов */
}

QRadioButton::indicator::checked {
    background-color: white; /* цвет индикатора при выборе */
}</string>
      </property>
      <property name="text">
       <string>Аннуитентный</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_diff">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: white;
}

QRadioButton::indicator {
    width: 10px;
    height: 10px;
    border: 2px solid white; /* цвет рамки */
    background-color: #333333; /* цвет фона индикатора */
    border-radius: 5px; /* скругление углов */
}

QRadioButton::indicator::checked {
    background-color: white; /* цвет индикатора при выборе */
}</string>
      </property>
      <property name="text">
       <string>Дифференцированный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>216</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_creditcalc">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: white;
}

QRadioButton::indicator {
    width: 10px;
    height: 10px;
    border: 2px solid white; /* цвет рамки */
    background-color: #333333; /* цвет фона индикатора */
    border-radius: 5px; /* скругление углов */
}

QRadioButton::indicator::checked {
    background-color: white; /* цвет индикатора при выборе */
}</string>
      </property>
      <property name="text">
       <string>Кредитный калькулятор</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_calc">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: white;
}

QRadioButton::indicator {
    width: 10px;
    height: 10px;
    border: 2px solid white; /* цвет рамки */
    background-color: #333333; /* цвет фона индикатора */
    border-radius: 5px; /* скругление углов */
}

QRadioButton::indicator::checked {
    background-color: white; /* цвет индикатора при выборе */
}</string>
      </property>
      <property name="text">
       <string>Калькулятор</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_depositcalc">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    color: white;
}

QRadioButton::indicator {
    width: 10px;
    height: 10px;
    border: 2px solid white; /* цвет рамки */
    background-color: #333333; /* цвет фона индикатора */
    border-radius: 5px; /* скругление углов */
}

QRadioButton::indicator::checked {
    background-color: white; /* цвет индикатора при выборе */
}</string>
      </property>
      <property name="text">
       <string>Депозитный калькулятор</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
